lib/configuration.ex

```diff
@@ -0,0 +1,89 @@
+defmodule Configuration do
+
+  def configure_game do
+    {player_1, player_2} = set_players()
+    board_size = get_board_size()
+    %Game{board: Board.create(board_size), 
+          size: board_size,
+          player_1: player_1,
+          player_2: player_2,
+          current_player: player_1}
+  end
+
+  def set_players(console \\ Console) do
```

Comment:
I was thinking about this function.  I believe we could probably use TDD sometime this week (or soon) to refactor it into a shape where it doesn't loop infinitely, which might make it easier to test.  We should all give it a look over tomorrow if possible!